ches/emailHandler.ts

```diff
@@ -93,3 +98,45 @@ export const sendEditRequestEmails = async (
     console.error('ERROR SENDING EDIT REQUEST EMAIL');
   }
 };
+
+export const sendDeleteRequestEmails = async (product: PrivateCloudRequestedProjectWithContacts, comment: string) => {
+  const email = render(DeleteRequestTemplate({ product }), { pretty: true });
+
+  try {
+    await sendEmail({
+      body: email,
+      to: [product.projectOwner.email, product.primaryTechnicalLead.email, product.secondaryTechnicalLead?.email],
+      subject: `${product.name} deletion request has been rejected`,
+    });
+  } catch (error) {
+    console.error('ERROR SENDING REQUEST REJECTION EMAIL');
+  }
+};
+
+export const sendDeleteRequestApprovalEmails = async (product: PrivateCloudRequestedProjectWithContacts) => {
+  const email = render(DeleteApprovalTemplate({ product }), { pretty: true });
+
+  try {
+    await sendEmail({
+      body: email,
+      to: [product.projectOwner.email, product.primaryTechnicalLead.email, product.secondaryTechnicalLead?.email],
+      subject: `${product.name} deletion request has been approved`,
+    });
+  } catch (error) {
+    console.error('ERROR SENDING REQUEST REJECTION EMAIL');
+  }
+};
+
+export const sendDeleteRejectionEmails = async (product: PrivateCloudRequestedProjectWithContacts, comment: string) => {
```

Comment:
should be sendRejectionEmails. Can also add the sendDeleteRequest emails in /api/rivate-cloud/delete

Reply:
sendRejectionEmails gets request as an argument and than productName: request.requestedProject.name, while sendDeleteRejectionEmails  gets product as an argument and than productName: product.name

Reply:
Would it be better if we make it just 1 function that takes string instead of product/request. So we just past the string as a parameter.

Reply:
it is impossible to pass only product name it also needs PO/TL emails